collector/time.go

```diff
@@ -34,18 +34,17 @@ func init() {
 // the current system time in seconds since epoch.
 func NewTimeCollector() (Collector, error) {
 	return &timeCollector{
-		metric: prometheus.NewCounter(prometheus.CounterOpts{
-			Namespace: Namespace,
-			Name:      "time",
-			Help:      "System time in seconds since epoch (1970).",
-		}),
+		desc: prometheus.NewDesc(
+			Namespace+"_time",
+			"System time in seconds since epoch (1970).",
+			nil, nil,
+		),
 	}, nil
 }
 
-func (c *timeCollector) Update(ch chan<- prometheus.Metric) (err error) {
+func (c *timeCollector) Update(ch chan<- prometheus.Metric) error {
 	now := float64(time.Now().Unix())
 	log.Debugf("Set time: %f", now)
-	c.metric.Set(now)
-	c.metric.Collect(ch)
-	return err
+	ch <- prometheus.MustNewConstMetric(c.desc, prometheus.CounterValue, now)
```

Comment:
It was a counter before.. But yeah I remember we discussed that timestamps should be gauges. Will change.